### PR TITLE
Feature/logs archive

### PR DESCRIPTION
**Pull request by j-fischer**
Created big object to archive log events based on threshold configuration. Log monitor now supports displaying messages from the archive as well.

### CODE REVIEW

**Comment by j-fischer** [comment]
@marcoalmodova Please have a look at this PR as it contains your confirmation-dialog component as discussed. Please let me know if you have any concerns or comments. 

Thanks again for your contribution!

File: rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+-->
+<!--
```
Comment (j-fischer) [inline rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.html:29]: @marcoalmodova I renamed the component as I made some small UI adjustments adding some spacing between the button. Added attribution and link to the original repository.

File: rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+/**
+ * Created by marcoalmodova on 23/02/2020.
```
Comment (j-fischer) [inline rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.js:30]: @marcoalmodova Same attribution here.

File: rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.html (hunk 1 above)
Comment (marcoalmodova) [reply rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.html:29]: Looks good to me, @j-fischer !

File: rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.js (hunk 2 above)
Comment (marcoalmodova) [reply rflib/main/default/lwc/rflibConfirmationDialog/rflibConfirmationDialog.js:30]: Looks good to me, @j-fischer !

**Pull request merged**